services/QuillLMS/app/models/concerns/public_progress_reports.rb

```diff
@@ -288,13 +288,21 @@ def get_time_in_minutes(activity_session)
     time > 60 ? '> 60' : time
   end
 
+  def format_direction(direction)
```

Comment:
Maybe add a comment about what this is doing, just because regexes are difficult to parse?